src/app/services/news.service.ts

```diff
@@ -1,57 +1,55 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Observable, from } from 'rxjs';
+import { Subject, Subscription } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { News } from '../models/News';
+import { RegionSelectService } from './region-select.service';
 
 export interface NewsService {
-  getNews(): Observable<News[]>;
+  getNewsObserver(): Subject<News[]>;
 }
 
 @Injectable({
   providedIn: 'root'
 })
-export class DummyNewsService implements NewsService {
-  getNews(): Observable<News[]>  {
-    return from([[
-      {
-        url: 'http://www.google.com',
-        title: 'COVID kills',
-        description: 'Dumb ppl still not wearing masks.'
-      },
-      {
-        url: 'http://www.bridgebase.com/v3',
-        title: 'Moron dies after injecting himself with bleach',
-        description: 'Trump claims injecting bleach as a COVID treatment is "very, very bad"'
+export class SmartableAiNewsService implements NewsService {
+  private url = 'https://api.smartable.ai/coronavirus/news/';
+  private subscriptionKey = '955eacf9525e45dd8d46a07b7daa649e';
+  private newsObserver: Subject<News[]>;
```

Comment:
small naming thing, subject is still an observable, not an observer. Anything that watches it would be the observer